src/network/NetworkAction.cpp

```diff
@@ -191,7 +191,7 @@ const std::vector<NetworkAction> NetworkActions::Actions =
     }, {
         STR_ACTION_KICK_PLAYER, "PERMISSION_KICK_PLAYER",
         {
-            GAME_COMMAND_KICK_PLAYER
+            -4
```

Comment:
@Broxzier The majority of the network commands have been negative thus far, i was pretty much just going with the flow